packages/taquito/src/operations/operations.ts

```diff
@@ -130,7 +135,7 @@ export class Operation {
     return (
       this.results.map((result) => {
         if (hasMetadataWithResult(result)) {
-          return result.metadata.operation_result.status;
+          return (result.metadata.operation_result as OperationResult).status;
```

Comment:
preference wise, i'd declare type defined variable and return variable.status then casting.

Reply:
same comment as the one above